Packs/Phishing/ReleaseNotes/2_2_7.md

```diff
@@ -0,0 +1,18 @@
+
+#### Playbooks
+##### Phishing Investigation - Generic v2
+- Use the "Process Microsoft's Anti-Spam Headers" sub-playbook in order to assess PCL/SCL/BCL scores.
+
+##### New: Process Microsoft's Anti-Spam Headers
+- This playbook will return SCL, BCL and PCL scores to the context (if exist).
+It will also do the following:
+1) Set the email classification to "spam" if the "SCL" score is equal or higher than 5.
+2) Set a value of "Medium" to a "MicrosoftHeadersSeverityCheck" field in the context if the PCL or BCL value is higher than 4.
```

Comment:
@tomer-pan This should be hardcoded? maybe worth to add a playbook input to define which number is equal to each value

Reply:
@David-BMS  These numbers are based on Microsoft's scoring as mentioned here: 
SCL - https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/spam-confidence-levels?view=o365-worldwide
BCL - https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/bulk-complaint-level-values?view=o365-worldwide
PCL - https://docs.microsoft.com/en-us/Exchange/antispam-and-antimalware/antispam-protection/antispam-stamps?view=exchserver-2019

Reply:
@David-BMS As we discussed, I added playbook inputs for setting a severity for each value.